pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -437,34 +500,16 @@ rec {
     };
   };
 
-  jsonedit = buildEclipsePlugin rec {
-    name = "jsonedit-${version}";
+  jsonedit = mirrorEclipsePlugins rec {
```

Comment:
As mentioned in the root comment of the PR, this change was needed due to peculiarities with the antlr dependency. I think it makes a good case that the new system is easier to use though!